Src/Particle/AMReX_Particles.H

```diff
@@ -229,14 +229,14 @@ public:
 #endif
                     ref_ratio.push_back(rr[i][0]);
                 }
-                return ref_ratio;  }() )
+                return ref_ratio;                  //!< }() )
```

Comment:
Seems that the script has a bug.

Reply:
Yes, the script I used definitely isn’t perfect - I tried to clean it up, but I’ve almost certainly missed a few things like this. I’ll take another pass through to see if I can spot the ones I missed